path/renamer.rkt

```diff
@@ -0,0 +1,21 @@
+#lang racket/base
+
+(provide auto-rename)
+
+(require racket/path
+         racket/system
+         framework/preferences)
+
+(define (auto-rename old-path new-path)
+  (define current-project-dir (preferences:get 'current-project))
+  (define (convert p) (find-relative-path current-project-dir p))
+  (recur-on-dir current-project-dir (convert old-path) (convert new-path))
```

Comment:
Do we only replace `old-path` with `new-path`? A problem that can see here is a module can be imported in two ways
1. `require "mod.rkt"`
2. `require collection/mod`

Current implementation didn't handle second one.

Reply:
how to ensure we get the correct collection name? If we are in a multi-collections project?